### ISSUE TITLE
yarn can't process packages with `@` in the name?

### ISSUE THREAD

**Issue by jasonbarry** [yarnpkg/yarn#2075]
Yarn version: 0.16.1
Node version:  6.5.0
Platform: darwin x64
OS: macOS 10.12

To reproduce: 
```
$ yarn add @blueprintjs/core
$ yarn outdated
```

results in 

```
error An unexpected error occured, please open a bug report with the information provided in "yarn-error.log".
```

yarn-error.log
```
Trace: 
  Error: https://registry.yarnpkg.com/@blueprintjs/core: Not found
      at Request.params.callback [as _callback] (/usr/local/lib/node_modules/yarn/updates/0.16.1/lib/util/request-manager.js:294:18)
      at Request.self.callback (/usr/local/lib/node_modules/yarn/updates/0.16.1/node_modules/request/request.js:187:22)
      at emitTwo (events.js:106:13)
      at Request.emit (events.js:191:7)
      at Request.<anonymous> (/usr/local/lib/node_modules/yarn/updates/0.16.1/node_modules/request/request.js:1048:10)
      at emitOne (events.js:96:13)
      at Request.emit (events.js:188:7)
      at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/yarn/updates/0.16.1/node_modules/request/request.js:969:12)
      at emitNone (events.js:91:20)
      at IncomingMessage.emit (events.js:185:7)
```

**Comment by wyze** [comment]
Please upgrade to the latest yarn (0.17.9) and see if that resolves your issue.

**Comment by jasonbarry** [comment]
How quickly I fall behind! Fixed as of 0.17.9; closing.
